src/styles/general.css

```diff
@@ -16,47 +35,12 @@ a {
   text-decoration: none;
 }
 
-.navBar {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  box-sizing: border-box;
-  padding: 0 40px;
-  height: 100px;
-  min-height: 100px;
-}
-
-.navLinks {
-  padding-right: 160px;
-}
```

Comment:
- [ ] Kindly it will be better 😃 if you align the navbar here with the content of the books store As you see here the navbar beginning and the content of data books are not aligned ,so please check the image below for more clarity 🚀💯


> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|  ![image](https://github.com/gisachris/react-bookstore-application/assets/83360636/f4085367-56f4-4071-9465-6e96c62fd0dd)  |    ![image](https://github.com/gisachris/react-bookstore-application/assets/83360636/13f65858-eaba-4b6a-918b-cec6cead51a6)   |